app/services/mapping_code_determiner.rb

```diff
@@ -53,8 +53,14 @@ def dispute_type_mapping_code
   def case_type_mapping_code
     # TODO: Add further when-branches once we have additional case types
     case tribunal_case.case_type
+    when CaseType::APPLY_DECISION_ENQUIRY
+      MappingCode::APPN_DECISION_ENQRY
```

Comment:
I know these abbreviations come out of an earlier commit, but I find them a bit jarring.  I'm assuming `APPN` is `Application`?  Or is it `Appellant`? Maybe not for this PR, but I'd like to see them expanded to their full terms.